calculate_trigger.py

```diff
@@ -26,15 +26,38 @@ def allign_fault_to_instruction(address, tbinfo_size, tbinfo_assembler, tbinfo_i
             return asm_addresses[i]
 
 
-def search_for_fault_location(
+def calculate_lifespan_from_start(
     filter_lists,
-    trigger_position,
     fault_address,
-    trigger_occurrences,
     goldenrun_tb_exec,
     goldenrun_tb_info,
+    trigger_occurrences,
+):
+    [idx, instruction_address] = find_fault(
+        fault_address, goldenrun_tb_exec, goldenrun_tb_info, trigger_occurrences
+    )
+    id_tb = goldenrun_tb_exec.at[idx, "tb"]
```

Comment:
I would suggest to name it `tb_id` like [here](https://github.com/Fraunhofer-AISEC/archie/blob/3b0bb56ad3ad15651c2fd226d4703a2d79ae053d/calculate_trigger.py#L73).

Reply:
Will do